snowflake-api/src/helpers.rs

```diff
@@ -0,0 +1,116 @@
+use crate::SnowflakeApiError;
```

Comment:
This should probably be called `upload_files.rs`, helpers could mean literally anything and obscures the purpose